ui/packages/consul-ui/app/routes/dc/nodes/show/index.js

```diff
@@ -7,6 +7,7 @@ export default class IndexRoute extends Route {
       .split('.')
       .slice(0, -1)
       .join('.');
+    model = this.modelFor(parent);
```

Comment:
what's this about?

Reply:
Oooooh I can't quite remember, lemme take a look 👀

Reply:
No idea, must have just looked weird to me when I was in here, both model coming from the arguments and model coming from parent are the same (as we have no model hook here I guess?).

Happy to leave in or take out really, I suppose leaving it in means that if we do add a model hook here, this hook still works as expected, so I'd if it was me I'd go with leaving it in. 🤷